starter/source/restart/ddsplit/c_dampvrel.F

```diff
@@ -0,0 +1,81 @@
+Copyright>        OpenRadioss
+Copyright>        Copyright (C) 1986-2024 Altair Engineering Inc.
+Copyright>
+Copyright>        This program is free software: you can redistribute it and/or modify
+Copyright>        it under the terms of the GNU Affero General Public License as published by
+Copyright>        the Free Software Foundation, either version 3 of the License, or
+Copyright>        (at your option) any later version.
+Copyright>
+Copyright>        This program is distributed in the hope that it will be useful,
+Copyright>        but WITHOUT ANY WARRANTY; without even the implied warranty of
+Copyright>        MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+Copyright>        GNU Affero General Public License for more details.
+Copyright>
+Copyright>        You should have received a copy of the GNU Affero General Public License
+Copyright>        along with this program.  If not, see <https://www.gnu.org/licenses/>.
+Copyright>
+Copyright>
+Copyright>        Commercial Alternative: Altair Radioss Software
+Copyright>
+Copyright>        As an alternative to this open-source version, Altair also offers Altair Radioss
+Copyright>        software under a commercial license.  Contact Altair to discuss further if the
+Copyright>        commercial version may interest you: https://www.altair.com/radioss/.
+Chd|====================================================================
+Chd|  W_DAMPING                       source/restart/ddsplit/w_seatbelts.F
+Chd|-- called by -----------
+Chd|        DDSPLIT                       source/restart/ddsplit/ddsplit.F
+Chd|-- calls ---------------
+Chd|        WRITE_DB                      source/restart/ddsplit/wrrest.F
+Chd|        WRITE_I_C                     ../common_source/tools/input_output/write_routtines.c
+Chd|        NLOCAL                        source/spmd/node/ddtools.F    
+Chd|        SEATBELT_MOD                  ../common_source/modules/seatbelt_mod.F
+Chd|====================================================================
+      SUBROUTINE C_DAMPVREL(DAMPR,IGRNOD,PROC,IDAMP_VREL,NDAMP_VREL_L)
+C-----------------------------------------------
+C   M o d u l e s
+C-----------------------------------------------
+      USE GROUPDEF_MOD     
+C---------------------------------------------
+C   I m p l i c i t   T y p e s
+C-----------------------------------------------
+#include      "implicit_f.inc"
+#include      "param_c.inc"
+#include      "com01_c.inc"
+#include      "com04_c.inc"
```

Comment:
not possible to add commons include anymore

Reply:
commons removed